cfme/v2v/infrastructure_mapping.py

```diff
@@ -216,7 +216,7 @@ class NetworkComponent(InfraMappingComponent):
     datastores = attr.ib(validator=attr.validators.instance_of(list))
     networks = attr.ib(validator=attr.validators.instance_of(list))
 
-    def fill_dict(self):
+    def fill_dict(self, target_selection=True):
```

Comment:
What is this for ? Target_selection ?
We are always selecting target .Aren't we ?

Reply:
by default we selects target but when we edit map we can not edit this selection

![Screenshot from 2019-07-16 13-08-55](https://user-images.githubusercontent.com/10824880/61275534-c9f53200-a7cb-11e9-9cd3-3be4bc07a011.png)